scalableminds/webknossos
# Check that sent json fields are json objects

### Steps to test:
- edit `TaskType Recommended Configuration`
- check pgadmin that the json field is a json object 

### Issues:
- fixes #3856 

------
